launch/__init__.py

```diff
@@ -0,0 +1,13 @@
+import launch.aws
+import test_util.aws
+
+
+def get_launcher(config):
+    """Returns the correct class of launcher from a validated launch config dict
+    """
+    platform = config['platform']
+    if platform == 'aws':
+        boto_wrapper = test_util.aws.BotoWrapper(
+            config['aws_region'], config['aws_access_key_id'], config['aws_secret_access_key'])
+        return launch.aws.AwsCloudformationLauncher(boto_wrapper)
+    raise launch.util.LauncherError('UnsupportedAction', 'Launch platform not supported: {}'.format(platform))
```

Comment:
This was just for keeping life simple while implementing the large swathes of code. Now that a lot more of this has been implemented, in the next iteration we should definitely add a launch.exceptions with 'UnsupportedActionError' and 'ValidationError' etc